examples/input-form/crate/src/lib.rs

```diff
@@ -26,6 +26,18 @@ impl Render for SayHelloTo {
         let input = Node::element(
             bump,
             "input",
+            [on(bump, "input", |root, vdom, event| {
+                let input = match event
+                    .target()
+                    .and_then(|t| t.dyn_into::<web_sys::HtmlInputElement>().ok())
+                {
+                    Some(input) => input,
+                    None => return,
```

Comment:
This may have just answered my previous question about when you wouldn't want to schedule a render

Reply:
;)